change/@azure-msal-browser-6911f6de-1b4b-48fc-9ac9-f298b28461bc.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix throw when attempting to getAccount before initialization",
```

```suggestion
  "comment": "Fix throw when attempting to getAccount before initialization #7720",
```
Is there a reason we stopped tagging the PR number in change files?

---

Thanks, I often create changefiles before opening the PR and forget to update once I have a number...we really should automate this 😉